src/app/controllers/concerns/permissions_concern.rb

```diff
@@ -12,6 +12,7 @@ def can?(user, model, action)
     case model
     when Function then function_can?(user, model, action)
     when Comment then comment_can?(user, model, action)
+    when Like then like_can?(user, model, action)
     else raise ObjectPermissionsMissing, model
     end
   end
```

Comment:
can we use pundit gem for permissions ?
to avoid these switch cases.

what is your opinion ?

Reply:
yeah I had to make this decision early when I was thinking about permissions and I thought that I really don't need a gem to do this implementation, this plain ruby and doesn't need any extra functionality that the gem will provide.
I will use switch if it means I get rid of an extra dependency, that's ok